specification/maps/resource-manager/Microsoft.Maps/stable/2017-01-01-preview/maps-management.json

```diff
@@ -784,6 +789,15 @@
           }
         }
       }
+    },
+    "MapsAccountProperties": {
```

Comment:
adding properties to existing version will be considered as breaking change. To double confirm, is this PR fixing swagger to align with existing service behavior? Does servicie already has this property returned?

Reply:
Yes this is to align with existing behavior, the service already has this property. But notice it's a readonly. So the GET->PUT->GET chain would not break. Here are existing approvals from specs-pr and the openApi Hub branch:

https://github.com/Azure/azure-rest-api-specs/pull/7428
https://github.com/Azure/azure-rest-api-specs-pr/pull/942